### PR TITLE
removed market from unsign

### PR DESCRIPTION
**Pull request by FredrikAreschoug**

### CODE REVIEW

**Review by johantjellden — APPROVED** [review]

File: member-service/src/main/java/com/hedvig/memberservice/services/ZignSecBankIdService.kt [hunk 1]
```diff
@@ -85,10 +85,10 @@ class ZignSecBankIdService(
 
     fun notifyContractsCreated(memberId: Long) = zignSecAuthentication.notifyContractsCreated(memberId)
 
-    private fun resolveTwoLetterLanguageFromMember(memberId: Long): String {
+    private fun resolveTwoLetterLanguageFromMember(memberId: Long): String = "DK"/* {
         val acceptLanguage = memberRepository.findById(memberId).get().acceptLanguage
         return getTwoLetterLanguageFromLocale(textKeysLocaleResolver.resolveLocale(acceptLanguage))
-    }
+    }*/
```
Comment (vonElfvin) [inline member-service/src/main/java/com/hedvig/memberservice/services/ZignSecBankIdService.kt:91]: ?

Comment (FredrikAreschoug) [reply member-service/src/main/java/com/hedvig/memberservice/services/ZignSecBankIdService.kt]: haha good catch testing zingsec stuff this morning

File: member-service/src/main/java/com/hedvig/memberservice/web/QualityAssuranceController.kt [hunk 2]
```diff
@@ -13,13 +12,11 @@ import org.springframework.web.bind.annotation.*
 class QualityAssuranceController(
     private val qualityAssuranceService: QualityAssuranceServiceImpl
 ) {
-    @PostMapping("{market}/unsignMember")
+    @PostMapping("unsignMember")
```
Comment (vonElfvin) [inline member-service/src/main/java/com/hedvig/memberservice/web/QualityAssuranceController.kt:15]: Is this fixed in back-office?

File: member-service/src/main/java/com/hedvig/memberservice/services/ZignSecBankIdService.kt (hunk 1 above)
Comment (FredrikAreschoug) [reply member-service/src/main/java/com/hedvig/memberservice/services/ZignSecBankIdService.kt]: haha good catch testing zingsec stuff this morning

File: member-service/src/main/java/com/hedvig/memberservice/web/QualityAssuranceController.kt (hunk 2 above)
Comment (FredrikAreschoug) [reply member-service/src/main/java/com/hedvig/memberservice/web/QualityAssuranceController.kt:15]: already done

**Pull request merged**